config/application.rb

```diff
@@ -35,6 +35,9 @@ class Application < Rails::Application
     # Configure sensitive parameters which will be filtered from the log file.
     config.filter_parameters += [:password, :password_confirmation]
 
+    # Localization
+    I18n.load_path += Dir["config/locales/*/*.yml"]
```

Comment:
`config/application.rb` is meant to configure [some determined settings](http://guides.rubyonrails.org/configuring.html#configuring-rails-components) **during** framework initialization, which might happen too early.

It would be safer to [wait for the framewrok to be loaded before altering it's configuration](http://guides.rubyonrails.org/configuring.html#using-initializer-files) in this case.

Reply:
You're certainly right, they use `config.i18n.load_path` in the [documentation](http://guides.rubyonrails.org/i18n.html). Even if the way I load the string is wrong, setting the path with `config.i18n.load_path` instead of `I18n.load_path` in application.rb should be good ?

Reply:
Using `config.i18n.load_path` looks good, yes.  I only expect `config.foo = :bar`  directives to appear in `config/application.rb` :wink: .